File: Tranbajo1/Codigo.py
def programa_principal():
    
    # Declaración e inicialización
    valores_numericos = [0] * 6
    
    # Asignación de valores iniciales
    valores_numericos[0] = 10
    valores_numericos[1] = 20
    valores_numericos[2] = 30
    valores_numericos[3] = 40
    valores_numericos[4] = 50
    valores_numericos[5] = 60
    
    # Modificación de un elemento
    valores_numericos[2] = 89
    
    # Visualización del contenido
    print("Contenido de la lista:")
    for indice in range(len(valores_numericos)):
        print(f"valores_numericos[{indice}] = {valores_numericos[indice]}")
    
    # Operación matemática
    resultado_suma = valores_numericos[0] + valores_numericos[2]
    print(f"\nResultado de valores numericos + valores numericos = {resultado_suma}")

# Ejecución del programa
    programa_principal()

File: Tranbajo1/Codigo.js
// Declaración
let numeros = new Array(5);

// Inicialización
numeros[0] = 10;
numeros[1] = 20;
numeros[2] = 30;
numeros[3] = 40;
numeros[4] = 50;

// Asignación
numeros[2] = 100;

// Prueba en pantalla
console.log("Contenido del arreglo:");
for (let i = 0; i < numeros.length; i++) {
    console.log(`numeros[${i}] = ${numeros[i]}`);
}

// Suma de dos elementos
let suma = numeros[0] + numeros[2];
console.log(`\nSuma de numeros[0] + numeros[2] = ${suma}`);

